specification/timeseriesinsights/resource-manager/Microsoft.TimeSeriesInsights/2017-02-28-preview/Swagger/timeseriesinsights.json

```diff
@@ -34,14 +34,42 @@
     }
```

Comment:
please don't change the folder structure! Note that this PR introduces a folder `Swagger` which also requires changing all the example paths - please adjust the structure to be consistent with the repo's guidelines. 🙂

Reply:
Reverted. For some reason I thought the swagger subfolder was required.